kubernetes/services/api-gateway.yaml

```diff
@@ -43,6 +38,18 @@ data:
     keystone.password=alcor_pass
     keystone.auth_type=password
     keystone.auth_url=http://192.168.10.10:5000/v3
+    neutron.url_prefix=/v2.0
+    logging.level.root=TRACE
+    
+    Spring.mvc.async.request-timeout=30000
+    server.max-http-header-size=65536
+    server.max-initial-line-length=65536
+    
+    spring.sleuth.sampler.probability=1.0
+    spring.application.name=api-gateway
+    # the zipkin host should be YOURS
+    spring.zipkin.base-url=http://10.213.43.187:9411
```

Comment:
@kevin-zhonghao @cj-chung Do we have zipkin conf in the application.properties files? We may need to remove them there and in the yaml files as well.

Reply:
We dont have zipkin conf in the application.properties files. It maybe added by Xiaodong

Reply:
I have removed them